server/views/pages/bodyscans/recordBodyScan.njk

```diff
@@ -36,7 +36,7 @@
                 <h2 class="govuk-heading-m">Date of body scan</h2>
                     {% set anotherDateHtml %}
                         {{ govukDateInput({
-                            id: "another-date-for-body-scan",
+                            id: "another-date",
```

Comment:
Removed `-for-body-scan` to fix anchor links for input fields as now the prefix of `another-date` will match to error names in this section